data/strings/strings.txt

```diff
@@ -22358,6 +22865,7 @@
     it:other = %d file trovati. Li vedrai dopo la conversione.
     ja = %d ファイルが見つかりました。 あなたは変換後にそれらを見るでしょう。
     ko = %d 파일을 찾았습니다. 회심 후 그들을 볼 수 있습니다.
+    lv = Lietotne atrada %d jauna(-as) datne(-es). Varat to (tās) apskatīt pēc konvertācijas.
```

Comment:
lv = Lietotne atrada %d jauna datne. Varat to apskatīt pēc konvertācijas.
lv:other = Lietotne atrada %d jaunas datnes. Varat tās apskatīt pēc konvertācijas.

Reply:
Latvian uses singular for all numbers, ending with 1, except 11.
Can this be expressed in the localisation framework here?

Reply:
Of course, [Unicode standard](https://www.unicode.org/cldr/charts/45/supplemental/language_plural_rules.html#lv) describes it in details and it is definitely supported in Android (not sure about other OSes). I will edit this right now @richlv

Reply:
Is Android using Unicode or GNU gettext standard for plurals? Or maybe doesn't matter in this case?

A correction in lv:one

lv:zero = Lietotne atrada %d jaunu datņu. Varat tās apskatīt pēc konvertācijas.
lv:one = Lietotne atrada %d jaunu datni. Varat to apskatīt pēc konvertācijas.
lv:other = Lietotne atrada %d jaunas datnes. Varat tās apskatīt pēc konvertācijas.